hypha/apply/activity/adapters/activity_feed.py

```diff
@@ -46,6 +46,7 @@ class ActivityAdapter(AdapterBase):
         MESSAGES.SKIPPED_REPORT: 'handle_skipped_report',
         MESSAGES.REPORT_FREQUENCY_CHANGED: 'handle_report_frequency',
         MESSAGES.BATCH_DELETE_SUBMISSION: 'handle_batch_delete_submission',
+        MESSAGES.BATCH_ARCHIVE_SUBMISSION: 'handle_batch_archive_submission',
```

Comment:
Hold onto merging this as https://github.com/HyphaApp/hypha/pull/2991 introduces changes in this.